Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "cuteness"
-version = "0.9.9"
+version = "1.0.0"
```

Comment:
We're not 1.0.0 yet, we cannot guarantee that stability

```suggestion
version = "0.9.10"
```

Reply:
sorry about that, incrementing versions is a bad habit of mine